packages/aws-cdk/test/test.docker.ts

```diff
@@ -77,7 +77,7 @@ export = {
       imageNameParameter: 'MyParameter',
       packaging: 'container-image',
       path: '/foo',
-      sourceHash: '1234567890abcdef',
+      sourceHash: '1234567890abcdefa-target',
```

Comment:
Can you please add a test case without a target?

Reply:
there are already tests in the file that assert source hash which would pick up any regressions?

Reply:
we usually like to add tests for specific new cases.

Reply:
@eladb not sure i follow, these aren't additive changes, rather modifications to prior behavior (thus updating existing tests). there are already tests that cover using docker without targets (targets were additive changes with their own tests in prior commits).

Reply:
ok got it. sorry, my misunderstanding.